src/UI/Components/Diagram/shapes.ts

```diff
@@ -23,17 +25,17 @@ export class ServiceEntityBlock extends shapes.standard.HeaderedRecord {
         attrs: {
           body: {
             class: "joint-entityBlock-body",
-            strokeWidth: 1,
+            strokeWidth: 0,
             cursor: "default",
           },
           header: {
-            class: "joint-entityBlock-header",
-            strokeWidth: 1,
+            strokeWidth: 0,
             cursor: "grab",
           },
           headerLabel: {
-            class: "joint-entityBlock-header-label",
+            fontFamily: `"RedHatMono", "Liberation Mono", consolas, "SFMono-Regular", menlo, monaco, "Courier New", monospace`,
```

Comment:
Do we really need the full track of different fonts?

Reply:
That's the same font track as in patternfly so I would opt for that to avoid some inconsistency just in case